### ISSUE TITLE
Running tests requires MASTODON_DOMAIN env var

### ISSUE THREAD

**Issue by knightpp** [kaimanhub/MastodonBot.ex#3]
See
```
$ mix test
** (System.EnvError) could not fetch environment variable "MASTODON_DOMAIN" because it is not set
    (elixir 1.17.3) lib/system.ex:705: System.fetch_env!/1
    /home/user/MastodonBot.ex/config/runtime.exs:8: (file)
    (stdlib 4.3.1.4) erl_eval.erl:748: :erl_eval.do_apply/7
    (stdlib 4.3.1.4) erl_eval.erl:961: :erl_eval.expr_list/7
    (stdlib 4.3.1.4) erl_eval.erl:290: :erl_eval.expr/6
    (stdlib 4.3.1.4) erl_eval.erl:282: :erl_eval.expr/6
    (stdlib 4.3.1.4) erl_eval.erl:961: :erl_eval.expr_list/7
```

**Comment by kaimanhub** [comment]
Fixed in scope of the https://github.com/kaimanhub/MastodonBot.ex/commit/59832e88e698367f07cb1bbb25ad95c86fa377fd

**Comment by knightpp** [comment]
Not fixed as of 59832e88e698367f07cb1bbb25ad95c86fa377fd
